core/src/main/java/io/kestra/core/validations/extractors/PropertyValueExtractor.java

```diff
@@ -0,0 +1,15 @@
+package io.kestra.core.validations.extractors;
+
+import io.kestra.core.models.property.Property;
+import io.micronaut.context.annotation.Context;
+import jakarta.validation.valueextraction.ExtractedValue;
+import jakarta.validation.valueextraction.ValueExtractor;
+
+@Context
+public class PropertyValueExtractor implements ValueExtractor<Property<@ExtractedValue ?>> {
+
+    @Override
+    public void extractValues(Property<?> originalValue, ValueReceiver receiver) {
+        receiver.value( null, originalValue.getValue());
```

Comment:
```suggestion
        receiver.value( null, originalValue.getExpression());
```

The value is computed late at rendering time

Reply:
The implementation is of course naive, I'll handle a better imp later